### PR TITLE
apkeep: init at 0.6.0

### PR DESCRIPTION
**Pull request by jyooru**
###### Motivation for this change

Add https://github.com/EFForg/apkeep

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -5776,6 +5776,16 @@
     githubId = 20658981;
     name = "Jarosław Wygoda";
   };
+  jyooru = {
```
Comment (legendofmiracles) [inline maintainers/maintainer-list.nix:5779]: Already added in https://github.com/NixOS/nixpkgs/pull/143831#pullrequestreview-799405913

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 144584` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>apkeep</li>
  </ul>
</details>

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -23771,6 +23771,8 @@ with pkgs;
 
   apache-directory-studio = callPackage ../applications/networking/apache-directory-studio {};
 
+  apkeep = callPackage ../tools/misc/apkeep/default.nix {};
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:23774]:
```suggestion
  apkeep = callPackage ../tools/misc/apkeep { };
```

File: pkgs/tools/misc/apkeep/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-RTzYvDkmE0kgl4FSOSjDuQ5G1E0ugFU41zVAMMroofM=";
+  };
+
+  cargoSha256 = "sha256-YFs2AOMGp0WNrceK14AnigZdJl+UsQdUchpxaI7HSXw=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/apkeep/default.nix:15]: You are sure this is not using the vendored openssl?

Comment (jyooru) [reply pkgs/tools/misc/apkeep/default.nix:15]: The build fails if I move `openssl` into `nativeBuildInputs`.

**Pull request merged**